RecoBTag/FeatureTools/interface/deep_helpers.h

```diff
@@ -102,5 +113,96 @@ namespace btagbtvdeep {
     VarInfo info(const std::string &name) const { return var_info_map.at(name); }
   };
 
+  int center_norm_pad(const std::vector<float> &input,
+                      float center,
+                      float scale,
+                      unsigned min_length,
+                      unsigned max_length,
+                      std::vector<float> &datavec,
+                      int startval,
+                      float pad_value = 0,
+                      float replace_inf_value = 0,
+                      float min = 0,
+                      float max = -1);
+
+  class ParticleNetConstructor {
+  public:
+    ParticleNetConstructor(const edm::ParameterSet &Config_,
```

Comment:
actually, if this class is just a constructor and nothing else, it doesn't need to be a class; it could just be a function in this namespace (the helper class for TritonGraphModules actually has some member variables and other functions)

Reply:
Yep, good point, thanks!